Number_types/include/CGAL/FPU.h

```diff
@@ -383,6 +383,22 @@ typedef unsigned int FPU_CW_t;
 #define CGAL_FE_TOWARDZERO   _RC_CHOP
 #define CGAL_FE_UPWARD       _RC_UP
 #define CGAL_FE_DOWNWARD     _RC_DOWN
+# elif defined  __arm__
```

Comment:
There are several possible FPUs for ARM (plus softfloat), and I think you don't have `__arm__` when using thumb, so I don't know if that is the right test.

Reply:
Thanks. I have fixed that in https://github.com/CGAL/cgal/pull/3805/commits/1aee92af751a7021c64082c69dfd2ca6c569c67c. For the moment, I only care about VFP and Neon.

Reply:
https://wiki.debian.org/ArmEabiPort suggests `#if (defined(__VFP_FP__) && !defined(__SOFTFP__))` like we already use elsewhere in the file.

Reply:
OK. Thanks for the reference. I have committed https://github.com/CGAL/cgal/pull/3805/commits/42cbce9fbf9ad3becb5d1c20ee2eb106b1de7c61.